demux/demux_lavf.c

```diff
@@ -1264,6 +1264,7 @@ static bool demux_lavf_read_packet(struct demuxer *demux,
     dp->duration = pkt->duration * av_q2d(st->time_base);
     dp->pos = pkt->pos;
     dp->keyframe = pkt->flags & AV_PKT_FLAG_KEY;
+    dp->is_wrapped_avframe = st->codecpar->codec_id == AV_CODEC_ID_WRAPPED_AVFRAME;
```

Comment:
is there any risk of this value not being reset in any codepath?

Reply:
I hope not.

Reply:
what about `demux_packet_unref_contents`?

Reply:
added check for that, it would fail anyway with the disk cache